### PR TITLE
improve build instruction and introduction

### PR DESCRIPTION
**Pull request by edouard-lopez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,15 +21,20 @@ Status
 
 Nethogs is a mature piece of software included in most Linux distributions.
 
-Ideas for features, as well as open bugs, can be found at https://github.com/raboof/nethogs/issues
+Ideas for features, as well as open bugs, can be found on  [issues' label:enhancement](https://github.com/raboof/nethogs/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement).
```
Comment (raboof) [inline README.md]: this'd only show the enhancements, not the bugs, right?

File: README.md [hunk 2]
```diff
@@ -18,15 +21,20 @@ Status
...
 
 ### Building from source
 
-Nethogs depends on ncurses, so you need to install the ncurses development
-libraries before building nethogs. For example on Debian, 'apt-get install
-libncurses5-dev'.
+Nethogs depends on `ncurses`, so you need to install the ncurses development
```
Comment (raboof) [inline README.md]: perhaps mention libpcap here, too

**Comment by raboof** [comment]
Nice, thanks, some minor comments

File: README.md (hunk 1 above)
Comment (edouard-lopez) [inline README.md]: Yes, it's a link to the issue page filtered by _enhancement_ label. You can this the [result link in my patch](https://github.com/edouard-lopez/nethogs/blob/patch-1/README.md).

**Comment by edouard-lopez** [comment]
It's updated

File: README.md (hunk 1 above)
Comment (raboof) [inline README.md]: But the text promises you'll also find 'open bugs' there, which you don't :).

**Pull request merged**